paulshryock/paul-shryock
# [#65]: Add Content Security Policy headers

## Summary
Add Content Security Policy headers.

## Testing
- Run `npm start`
- See that CSP headers are served in production
- Equivalent CSP meta elements are served in development

## Issue(s)

Closes #65

## Changelog

### Added
- Add Content Security Policy headers. [#65]

## Release Checklist
